duo-client/src/main/java/com/duosecurity/client/Http.java

```diff
@@ -127,7 +127,7 @@ public Object executeJSONRequest() throws Exception {
     JSONObject result = new JSONObject(executeRequestRaw());
     if (!result.getString("stat").equals("OK")) {
       throw new Exception("Duo error code ("
-          + result.getInt("code")
+          + result.get("code").toString()
```

Comment:
There was little point in converting it to an int when we then just turned it into a string for the error message - so just let it be whatever it is and stringify it.